art12/templates/summary/eu_map.html

```diff
@@ -0,0 +1,9 @@
+{% block content %}
+  <body>
+    {% if map_exists %}
+      <img src="{{ MAPS_URI }}{{ speciescode }}{{ suffix }}.png" height="780">
```

Comment:
Send the computed URI from the view code, i.e. put it in the `map_exists` variable.